lua/nvim-tree/explorer/init.lua

```diff
@@ -665,8 +669,26 @@ function Explorer:get_nodes()
   return self:clone()
 end
 
+---@param node Node
+function Explorer:dir_up(node)
+  if not node or node.name == ".." then
+    require("nvim-tree.explorer.change-dir").fn("..")
+  else
+    local cwd = core.get_cwd()
+    if cwd == nil then
+      return
+    end
+    local newdir = vim.fn.fnamemodify(utils.path_remove_trailing(cwd), ":h")
+    require("nvim-tree.explorer.change-dir").fn(newdir)
+    require("nvim-tree.actions.finders.find-file").fn(node.absolute_path)
+  end
+end
+
+Explorer.change_dir = change_dir
```

Comment:
`Explorer.change_dir` is assigned to the `Explorer` class, instead of the instance. This isn't great as all `Explorer` instances will share the same `change_dir`.
It should be set to the instance during the constructor e.g. `self.change_dir = ...`